workspace/models.py

```diff
@@ -1,5 +1,5 @@
 from django.db import models
-from django.conf import settings
+from xlack import settings
```

Comment:
Had you seperated settings path to avoid confilct?

Reply:
you can see `settings.py` in `xlack` directory.

Reply:
yep. i had seen so i asked you about the purpose

Reply:
The reason I wrote that is I want to avoid conflict between default user model, but now seems "don't needed to be".

Whenever I change that and migrate that, django don't recognize it's difference.

Reply:
I see I just want to know the reason